### ISSUE TITLE
Screen reader does not announce status messages for empty lists

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#80]
**User Experience:**
Users using screen reader would not able to get acknowledgement and user will get confused if it is not announced by it. Screen reader should announce about status message.

Note: User credentials should NOT be included in the bug.
 
Repro steps:
1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add section.
7. Select various type of section in the dialog for layout.
8. Enable screen reader and navigate to "Form Library" button and select it.
9. Verify the issue.
 
**Actual Result:**
Screen reader does not announce status message as ‘It seems that your library is empty’ which is appears after activating ‘Form Library’ tab.

**Expected Result:**
Screen reader should announce status message as ‘It seems that your library is empty’ which is appears after activating ‘Form Library’ tab.

**Note:**
Similar issue exist with Chrome + NVDA, Chrome + JAWS\
Similar issue exist for all tab items present in page.

**MAS Reference:**
MAS 4.1.3 - Status Messages ***

**Reference Links:**
Accessibility Insights! - Identify accessibility bugs before check-in and make      bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries
 
![image](https://user-images.githubusercontent.com/2320302/76659592-14ce7180-6534-11ea-966e-c9ad425e2b70.png)
